ipatests/test_integration/test_installation.py

```diff
@@ -1190,6 +1190,21 @@ def run_request(url, expected_stdout=None, expected_stderr=None):
                 expected_stdout=f'href="https://{self.master.hostname}/'
             )
 
+    def test_pac_configuration_enabled(self):
+        """
+        This testcase checks that the default PAC type
+        is added to configuration.
+        """
+        base_dn = str(self.master.domain.basedn)
+        dn = DN(
+            ("cn", "ipaConfig"),
+            ("cn", "etc"),
+            base_dn
+        )
+        result = tasks.ldapsearch_dm(self.master, str(dn),
+                                     ["ipaKrbAuthzData"])
+        assert 'ipaKrbAuthzData: MS-PAC' in result.stdout_text
```

Comment:
This test validates that the value is set for new installations. We also need to add a test simulating an update from a deployment without the value:
- install IPA
- delete the attribute ipaKrbAuthzData: MS-PAC
- call ipa-server-upgrade
- check that the value is added
A good place for this new test would be in test_upgrade.py.

Can you also add "Related: https://pagure.io/freeipa/issue/9632" to you commit message?